src/TwigEnvironmentFactory.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace Zend\Expressive\Twig;
+
+use ArrayObject;
+use Interop\Container\ContainerInterface;
+use Twig_Environment as TwigEnvironment;
+use Twig_Extension_Debug as TwigExtensionDebug;
+use Twig_ExtensionInterface as TwigExtensionInterface;
+use Twig_Loader_Filesystem as TwigFilesystem;
+use Twig_Loader_Filesystem as TwigLoader;
```

Comment:
I probably copy pasted it from somewhere. Good catch tough. I guess the TwigLoader will do for both.